addons/skin.estuary/xml/Variables.xml

```diff
@@ -383,7 +383,9 @@
 		<value>$LOCALIZE[19021] / $INFO[Control.GetLabel(29)] - $INFO[Control.GetLabel(30)]</value>
 	</variable>
 	<variable name="BreadcrumbsPVRRecordingsVar">
-		<value condition="Window.IsActive(TVRecordings)">$LOCALIZE[19020] / $LOCALIZE[19017]$INFO[Control.GetLabel(30), / ]</value>
+		<value condition="Window.IsActive(TVRecordings) + String.Contains(Control.GetLabel(7),*)">$LOCALIZE[19020] / $LOCALIZE[19017]$INFO[Control.GetLabel(30), / ] - $LOCALIZE[19179]</value>
+		<value condition="Window.IsActive(TVRecordings) + !String.Contains(Control.GetLabel(7),*)">$LOCALIZE[19020] / $LOCALIZE[19017]$INFO[Control.GetLabel(30), / ]</value>
+		<value condition="Window.IsActive(RadioRecordings) + String.Contains(Control.GetLabel(7),*)">$LOCALIZE[19021] / $LOCALIZE[19017]$INFO[Control.GetLabel(30), / ] - $LOCALIZE[19179]</value>
```

Comment:
Out of curiosity: Can you explain this `GetLabel(7)`voodoo to me? Is this a special label and if yes, what is its purpose and where does its value come from?

Reply:
Used to determine if a radio button is on or off.

When on they will return `*`

Reply:
7 is the ID of the radio button for deleted recordings. It comes from Includes_MediaMenu.xml.

Reply:
Ah, thanks. Seems a bit hacky to me, but my skinning skills a very basic.